LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -294,39 +295,83 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                      </decoNote>
+                      <decoNote xml:id="d4" type="band">
+                          <locus target="#194va"/>
+                          <desc>Humble band by a later hand.</desc>
+                      </decoNote>
                   </decoDesc>
-                        
-                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#194vb"/>
+                                    <desc type="ProtectivePrayer">Erased and partially illegible text by a poorly trained hand with magical and protective 
+                                        content: <foreign xml:lang="gez"><gap reason="illegible" unit="lines" quantity="13"/>ታዝዕግ፡ እኩይ፡ የግ፡ አምሐልኩከ፡ 
+                                                በእግዚአብሔር፡ አብ፡ ወክርስቶስ፡ ወልድ፡ ሣልሳዊ፡ መንፈስ፡ ቅዱስ፡ ሥርናይየ፡ ውስተ፡ ከሉ፡ አካልየ፡ ኢትባዕ።</foreign></desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="194vb" to="195ra"/>
+                                    <desc type="ProtectivePrayer">Text by a poorly trained hand with magical and protective content: 
+                                        <foreign xml:lang="gez">ስቡሕመደልው፡ ተደለው፡ ካፈር፡ ልውያንተድይንተምያን፡ በጊዚአምናት፡ እምለባውዝ፡ ልበ፡ በረዛዝ፡ ኡግጹግ፡ 
+                                            <gap reason="ellipsis" unit="lines" quantity="8"/>ዋላሰብእ፡ እኪተ፡ ሚካኤልበእንአነ፡<pb next="195ra"/>ሰአልምሕርብ፡ 
+                                            ብርያል፡ ያመጽዕ፡ ዋግ፡ በሰብእ፡ እኪተ፡ በሰአብኤሎስ። ኤጊጊዚ፡ በል፡ ዋግ፡ ዋግ፡ ዋግ፡ <gap reason="ellipsis" unit="lines" quantity="4"/>
+                                            ወልድ፡ ሣልሳዊ፡ መንፈሰቅዱስ፡ ሥርናይየ፡ ውስተኲሉ፡ ኀበእክልየ፡ ኢትባዕ፡</foreign></desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1r #195v"/>
+                                    <desc type="StampExlibris">Small round stamp with coat of arms, lion, unicorn and inscription: 'British Museum'.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#5v #9v #15v #21v #27v"/>
+                                    <desc type="StampExlibris">Small round stamp with crown and inscription: 'British Museum' on every fourth to every sixth folio.</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#2r"/>
+                                    <desc type="OwnershipNote">Note by a later Ethiopian hand: <foreign xml:lang="gez">
+                                        <ref target="LIT1672Isaiah">ኢሳይያስ፡</ref> ዘቅዱስ፡ <placeName ref="INS0101MadhaneAlam">መድኃኒ፡ ዓለም</placeName>።
```

Comment:
I think I would not mark up the work ID here, as the Ethiopic traditional labels do not coincide with CAe IDs. In this case the label clearly is attached to the whole collection. Just like the label Afawarq may mean any of the collections of biblical commentaries or of homilies.

Reply:
Ok. I see, it is not neccessary to encode the work ID here.